FieldKit/app/services/db-interface.js

```diff
@@ -305,29 +308,71 @@ export default class DatabaseInterface {
         });
     }
 
-    _updateStream(station, status, type, index) {
-        if (!station.deviceId && station.device_id) {
-            station.deviceId = station.device_id;
+    getStationByStationIds(ids) {
```

Comment:
I believe this should be getStreamsByStationIds

Reply:
Yep